README.md

```diff
@@ -1,2 +1,6 @@
 # cell_counting
 We create machine learning algorithm to count cells in the microscopy photo
+
+
+## https://www.jetbrains.com/pycharm/download/#section=windows
+Choose your Operation System (Windows, macOS, Linux). Click „Download” (Community – free, open-source version). Double click – open the PyCharm (.exe file) to open PyCharm Community Edition Setup. Click „next” , then click „next” for the second time. Check checkboxes:„64-bit launcher”, „ Add Open Folder as Project”, „Download and install JRE x86 by JetBrains”. Click „Next” and click „Install”. Click „Run PyCharm Community Edition” and subsequently click „Finish” (PyCharm installed, shortcut on desktop created). Check „Do not import settings” in Complete Installation window and click „Ok”. Check „I agree that I have read and accept the terms of this User Agreement” checkbox and click „Continue” in JetBrains Privacy Policy window. Choose „Don’t send” in Data Sharing window. Set a favourite UI theme (Darcula/Light). Click „Next: Featured Plugins”. Click „Start using PyCharm”.
```

Comment:
Could you write it as a list of points, rather than a large chunk of text?
Can I connect the github project while configuring pycharm? If so, how?

Reply:
hmm so should I create a new version and new request using PyCharm or somewhere here?

Reply:
that's a very good question! There are 2 approaches - the first one, which is also simpler, is to add a new commit with your changes and push it to this branch (so the same as before, just 'push' instead of 'create pull request'. The second option, which is a bit more complicated is to make your changes, add them, an then instead of making a new commit, you amend your changes to the previous commit. From the command line you do `git commit --ament` rather than `git commit -m 'asdasd'` - I'm not sure how to do it in pycharm - you'd have to play about a bit.
 I personally prefer the second way, as otherwise there are a lot of commits that are just fixes, which makes it hard to read the history. For now either is ok - when you're most used to git then you can start to worry about the history, rebases and overwriting branches :P

Reply:
there's no need to make a new pull request - if you push changes to this branch, then the pull request will update itself

Reply:
lol I already read it, thank you!
I made it somehow by myself before reading, probably in the way similar to the one you described as the simply one, sorry if it's little bit messy 🐇